### PR TITLE
TSK-671: Allow upload of large classification or workbasket files

### PR DESCRIPTION
**Pull request by BVier**
Allow larger uploads (up to 10 MB).
Additionally corrected a typo.

### CODE REVIEW

File: rest/taskana-rest-spring-example/src/main/resources/application.properties [hunk 1]
```diff
@@ -49,6 +49,9 @@ taskana.ldap.groupsOfUser=memberUid
 taskana.jobscheduler.async.cron=0 * * * * *
 ####### cache static resources properties
 spring.resources.cache.cachecontrol.cache-private=true
+####### for upload of big workbasket- or classification-files
+spring.servlet.multipart.max-file-size=10MB
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-example/src/main/resources/application.properties]: is 10mb a reasonable size? I thing 100mb or something around there is more reasonable.

Comment (BVier) [reply rest/taskana-rest-spring-example/src/main/resources/application.properties]: The whole classification file for the db test-data is about 1,5MB.
So I thought 10 MB are more than enough space for at least this case.

The workbasket file in our db is even smaller, but probably it's bigger in the real use case.
Still it should not be bigger than 10MB in my opinion.

But of course we don't know how the file size will change for future use cases.
In this case the admin could simply change this file.

Comment (BVier) [reply rest/taskana-rest-spring-example/src/main/resources/application.properties]: But if there are no security concerns we could set the file-size to '-1' and we don't have to decide, what the 'real' maximum size is.

Comment (holgerhagen) [reply rest/taskana-rest-spring-example/src/main/resources/application.properties]: It's a good practice to restrict the size. Let's stick to the 10 MB and see how far we can get with it.

**Pull request merged**